tests/test_great_expectations.py

```diff
@@ -322,12 +322,14 @@ def test_stats_mixed_expectations():
 
 
 class TestIO(unittest.TestCase):
+    @pytest.mark.filesystem
     def test_read_csv(self):
```

Comment:
```suggestion
@pytest.mark.filesystem
class TestIO(unittest.TestCase):
    def test_read_csv(self):
```

@Shinnnyshinshin you can apply a marker to a test class and it will apply to all it's testing methods.
https://docs.pytest.org/en/7.1.x/example/markers.html#marking-whole-classes-or-modules

Then you can remove all the other marks.

Reply:
I thought of this @Kilo59, but not all of the tests in this class have the same marker. There are ones with pyarrow etc.

Reply:
otherwise, yes I think the `filesystem` marker could have been applied